include/rapidcheck/detail/Unicode.hpp

```diff
@@ -0,0 +1,96 @@
+#pragma once
+
+namespace rc {
+namespace detail {
+
+template<typename T, typename RandomType>
```

Comment:
Yes, 21 bits is currently all that is needed for Unicode. The algorithm currently favors codepoints with a low value by a significant margin (roughly 50%), though still about 12% chance to pick one of the very high values. It does not take the size attribute into account for individual characters, just full strings. But perhaps it should?